tools/BugReportTool/BugReportTool/InstallationFolder.cpp

```diff
@@ -106,7 +106,8 @@ wstring GetChecksum(path filePath)
 		return L"CryptCreateHash() failed. " + get_last_error_or_default(GetLastError());
 	}
 
-	while (bResult = ReadFile(hFile, rgbFile, bufferSize, &cbRead, NULL))
+	bResult = ReadFile(hFile, rgbFile, bufferSize, &cbRead, NULL);
+	while (bResult)
```

Comment:
This change looks wrong. It looks like an infinite cycle like this since bResult wouldn't be recalculated.

Reply:
Remove the argument when it was unused following the [C++ core guidelines](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#f9-unused-parameters-should-be-unnamed) maybe_unused is needed because of the #if.
I comment the argument name because the comment in the code refer to its name, it would have been confused to remove it completely.
This change is part of the https://github.com/microsoft/PowerToys/pull/21434 PR, for the warning 4706 (assignment within conditional expression) i expect it to be merged first but I want to try the changes together in the CI